File: vb #old/LANIP/Form1.vb
Imports System.Net
Public Class Form1
    Dim i As Integer
    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Try
            ListBox1.Items.Add("Hostname: " & Dns.GetHostName)
            Console.WriteLine("Hostname: " & Dns.GetHostName)
            ListBox1.Items.Add("----------------------------------------------------------------------")
            Console.WriteLine("----------------------------------------------------------------------")
            For index = 1 To 99
                ListBox1.Items.Add(CType(Dns.GetHostEntry(Dns.GetHostName).AddressList.GetValue(i), IPAddress).ToString)
                Console.WriteLine(CType(Dns.GetHostEntry(Dns.GetHostName).AddressList.GetValue(i), IPAddress).ToString)
                i = i + 1
                Status.Text = "Grabbed " & i & " IPs"
            Next
        Catch ex As Exception
        End Try
    End Sub

    Private Sub ListBox1_SelectedIndexChanged(sender As Object, e As EventArgs) Handles ListBox1.SelectedIndexChanged
        Try
            Clipboard.SetText(ListBox1.SelectedItem)
            Status.Text = "Copied " & Clipboard.GetText & " to clipboard"
        Catch ex As Exception
        End Try
    End Sub
End Class

File: EzIP/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.Net;

namespace EzIP
{
    public partial class Form1 : Form
    {
        public static string line = "-----------------------------------------------------------------------------------";

        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            toolLbl.Text = "Click on an IP to copy to clipboard";
            init();
        }
        private void init()
        {
            lstBox.Items.Add("The IPs of all of your adapters");
            lstBox.Items.Add("Click on one to copy to clipboard");
            lstBox.Items.Add(line);

            runIPGrab();
        }
            private void runIPGrab()
            {

                try
                {
                    //Add the computers name to the listbox
                    lstBox.Items.Add("Hostname: " + Dns.GetHostName());
                    lstBox.Items.Add(line);

                    string ip;
                    IPHostEntry host = default(IPHostEntry); //host ip adapters
                    string hostname;
                    hostname = System.Environment.MachineName; //host's computer
                    host = Dns.GetHostEntry(hostname); //checking the dns entries on the computer
                    foreach (IPAddress IP in host.AddressList) //for every ip adapter in host computer...
                    {
                        if (IP.AddressFamily == System.Net.Sockets.AddressFamily.InterNetwork) //if the ip = ipv4
                        {
                            ip = Convert.ToString(IP); //store in a string
                            lstBox.Items.Add(ip); //output to listbox
                            lstBox.Items.Add(line);
                        }
                    }

                }
                catch (Exception ex)
                {
                    MessageBox.Show(ex.Message);
                    throw;
                }
            }

        private void lstBox_SelectedIndexChanged(object sender, EventArgs e)
        {
            try
            {
                if (!(lstBox.SelectedItem as string == line)){
                Clipboard.SetText(Convert.ToString(lstBox.SelectedItem));
                toolLbl.Text = ("Copied " + lstBox.SelectedItem + " to clipboard");
                }
            }
            catch (Exception ex)
            {
                toolLbl.Text = ex.Message;
            }
        }
    }
}
